package.json

```diff
@@ -29,13 +30,22 @@
     "concurrently": "^2.2.0",
     "eslint": "^3.2.2",
     "eslint-config-d-centralize": "^0.1.0",
+    "jasmine": "^2.5.1",
     "jasmine-ajax": "^3.2.0",
+    "jasmine-as-promised": "0.0.8",
     "jasmine-core": "^2.4.1",
+    "jasmine-node": "^1.14.5",
+    "jasmine-promises": "^0.4.1",
     "karma": "^1.2.0",
     "karma-browserify": "^5.1.0",
     "karma-cli": "^1.0.1",
+    "karma-coverage": "^1.1.1",
     "karma-jasmine": "^1.0.2",
     "karma-phantomjs-launcher": "^1.0.1",
-    "watchify": "^3.7.0"
+    "mock-socket": "^5.0.1",
+    "q": "^1.4.1",
+    "sinon": "^1.17.6",
+    "watchify": "^3.7.0",
+    "xhr-mock": "^1.6.0"
```

Comment:
From the name it seems that this is a package that does what jasmine-ajax already does.